src/dbnode/client/write_state.go

```diff
@@ -199,6 +237,16 @@ func (w *writeState) completionFn(result interface{}, err error) {
 	w.decRef()
 }
 
+func (w *writeState) setHostSuccessList(hostID, pairedHostID string) {
+	w.reusableByteID.Reset(ident.StringID(pairedHostID).Bytes())
+	if findHost(w.hostSuccessList, w.reusableByteID) {
+		w.success++
+		w.successAsLeavingAndInitializingCountTowardsConsistency = true
```

Comment:
wait - why are you setting this bool? isn't this how you control whether we should enable your new code path?

Reply:
you're only using this bool to decide whether you should emit the separate metric indicating the new path was used. call it something which indicates that. `initializeLeavingPairCounted` or something